crates/beerus-rpc/src/server.rs

```diff
@@ -233,6 +237,18 @@ impl BeerusApiServer for BeerusRpc {
             .await
             .unwrap())
     }
+
+    async fn starknet_pending_transactions(&self) -> Result<Vec<Transaction>> {
+        let transactions = self
+            ._beerus
+            .starknet_lightclient
+            .pending_transactions()
+            .await
+            .unwrap();
```

Comment:
Needs proper implementation of error handling.

Reply:
Noted